edn/tests/tests.rs

```diff
@@ -33,6 +33,12 @@ fn k_plain(name: &str) -> Value {
     return Keyword(symbols::Keyword::new(name));
 }
 
+// Helper for making wrapped symbols with a namespace
```

Comment:
Feel like we can clean up a lot of these tests in a follow up! This file is large.